entries/m/medium.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Medium": {
+    "domain": "medium.com",
+    "contact": {
+      "email": "yourfriends@medium.com"
```

Comment:
I can add the Twitter handle, but the reason I initially didn't is because every time someone shares a post from Medium on Twitter, it adds "via @medium" at the end. Medium very rarely replies to their messages (only from authors they're amplifying) and doesn't have DMs enabled. For that reason, I don't think that they read their mentions. Let me know your thoughts.

Reply:
Okay let's leave it out then.